server.go

```diff
@@ -470,7 +470,7 @@ func (s *Server) deregisterActor(nsName string) {
 			cancel()
 			return err != nil
 		})
-	if err != nil {
+	if err != nil && err != registry.ErrNotOwner {
```

Comment:
```suggestion
	// Ingnore ErrNotOwner because most likely the previous owner panic'ed or exited badly. 
	// So we'll ignore the error and let the mailbox creator retry later.  We don't want to panic
	// in that case because it will take down more mailboxes and make it worse. 
	if err != nil && err != registry.ErrNotOwner {
```